WordPressKit/WordPressOrgXMLRPCApi.swift

```diff
@@ -254,6 +282,33 @@ open class WordPressOrgXMLRPCApi: NSObject {
         return progress
     }
 
+    /// Call an XMLRPC method.
+    ///
+    /// ## Error handling
+    ///
+    /// Unlike the closed-based APIs, this method returns a concrete error type. You should consider handle the errors
+    /// as they are, instead of casing them to `NSError` instance. But in case you do need to cast them to `NSError`,
+    /// considering using the `asNSError` function if you need backward compatiblity with existing code,
```

Comment:
```suggestion
    /// Unlike the closure-based APIs, this method returns a concrete error type. You should consider handling the errors
    /// as they are, instead of casting them to `NSError` instance. But in case you do need to cast them to `NSError`,
    /// considering using the `asNSError` function if you need backward compatibility with existing code.
```

_I only noticed the typo in compatibility when I loaded the text in this `textarea` in GitHub and the spell checker picked it up._

Reply:
🙈 Thanks! Updated in https://github.com/wordpress-mobile/WordPressKit-iOS/pull/719/commits/bcf8ea9e81da5b5471411cf25c879b6b252efc82